### DISCUSSION TITLE
Search doesn't work

### DISCUSSION THREAD

**Discussion by nicfab** [adityatelange/hugo-PaperMod#731] [Q&A] [answered]
Hi!
I set up a page “Search” on my microblog that I am testing.
However if I input any character, the search doesn’t work.
The `config.toml` contains the following lines:

```
[[languages.en.menu.main]]
    name = "Search"
    pageref = "page/search"
    weight = 4

```
…

```
[[languages.it.menu.main]]
    name = "Search"
    pageref = "page/search"
    weight = 4

```
and the following:

```
[params.fuseOpts]
  isCaseSensitive = false
  shouldSort = true
  location = 0
  distance = 1_000
  threshold = 0.4
  minMatchCharLength = 0
  keys = [ "title", "permalink", "summary", "content" ]

[outputs]
home = [ "HTML", "RSS", "JSON" ]
```

I created two pages that are respectively under `content/english/page` and under `content/italian/page` with the following declaration:

```
---
title: "Search" # in any language you want
layout: "search" # is necessary
# url: "/archive"
# description: "Description for Search"
summary: "search"
placeholder: "placeholder text in search input box"
---

```
Where is the issue?

**Comment by nicfab** [comment, marked as answer]
Hi!
I solved the arcane!
According to what I read here https://github.com/adityatelange/hugo-PaperMod/tree/exampleSite#directory-tree, it seems there is an issue related to the PaperMod configuration of the **directory tree**.

I was trying to keep organized both pages and posts, creating the following folders:
`/content/english/page`
`/content/english/post`
`/content/italian/page`
`/content/italian/post`
However, with this setting, the search doesn't work.

So, I moved all the pages files under `/content`, distinguishing the Italian ones as filename.it.md.
Thus, I created two folders and precisely `/content/post` and `/content/post_it` where I saved the English and the Italian, respectively.
In this way, the search works properly.
Anyway, if I am wronging or there is another solution to keep the files organised, please let me know.

**Comment by yvoictra** [comment]
Hello,

I have another configuration which seeems better to me.

```
/content/posts/<post-name>/index.en.md
/content/posts/<post-name>/index.es.md
```

So I have in the same folder the 2 languages of the post. Also, search function works great.

Best Regards.
